src/main/scala/com.snowplowanalytics.snowplow.micro/ConfigHelper.scala

```diff
@@ -138,10 +162,19 @@ private[micro] object ConfigHelper {
         throw new IllegalArgumentException(s"Error while reading Iglu config file: $e.")
     }
 
-    (
+    val enrichmentDirectory = Option(getClass.getResource("/enrichments"))
+      .fold(Paths.get("."))(url => Paths.get(url.toURI))
```

Comment:
Is the fallback `Paths.get(".")` here to help developers of Micro, or to help users of Micro?  As a developer, I think it would annoy me to have Micro reading files I didn't want it to read.  But if you've added it here because you think it will help users, then I'm OK with it.

Reply:
It’s for developers, yes. I have to put some path in, otherwise it’s impossible to test this via sbt. (You would think that adding test enrichments to `src/main/resources/enrichments` would work, but it doesn’t because it’s in a jar and `Path#listFiles` fails.)

Reply:
If it's just for sbt, can you add them to the lib directory? https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html

Reply:
I made it so that you can add to `./config/enrichments`, which mirrors the Docker setup nicely. That also allows to test embedded schemas by putting them into `./config/iglu-client-embedded` by the way.